### ISSUE TITLE
README.rst has text that causes encoding issues

### ISSUE THREAD

**Issue by pham7quang** [Bouke/django-two-factor-auth#360]
When pulling the latest `master` of this repo as a python package with pip, I get an encoding error:
```
Obtaining two_factor from git+https://github.com/Bouke/django-two-factor-auth#egg=two_factor (from -r requirements.txt (line 23))
  Cloning https://github.com/Bouke/django-two-factor-auth to /two-factor
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/src/two-factor/setup.py", line 7, in <module>
        long_description=open('README.rst').read(),
      File "/usr/lib/python3.6/encodings/ascii.py", line 26, in decode
        return codecs.ascii_decode(input, self.errors)[0]
    UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 1097: ordinal not in range(128)
```

## Expected Behavior
The weirdly encoded character `⚠️` should be removed from the `README.rst`

## Possible Solution

In the latest commit 3f1b8f5, the odd character was added.  Please remove if possible or use an ascii friendly character.

**Comment by moggers87** [comment]
This works for me on Python 3.8 and setuptools 41.6.0. Could you tell us which version of setuptools you're using?

**Comment by moggers87** [comment]
And it works on Python 3.6 too.

Also I notice that your Python install appears to be defaulting to ASCII rather than UTF-8. Do you have any why that would be?

**Comment by Bouke** [comment]
[Python suggests](https://packaging.python.org/guides/making-a-pypi-friendly-readme/) to do it this way:

```py
from setuptools import setup

# read the contents of your README file
from os import path
this_directory = path.abspath(path.dirname(__file__))
with open(path.join(this_directory, 'README.md'), encoding='utf-8') as f:
    long_description = f.read()

setup(
    # ...
    long_description=long_description,
    long_description_content_type='text/x-rst'
)
```

@pham7quang  can you verify that with this changes it works for you?

**Comment by pham7quang** [comment]
@Bouke @moggers87 - thanks for getting back to me so quickly. I found a solution on my end, thought I would share.

**Current Settings:**
This is my pip and python version:
```
root@d1a4620dbff1:/usr/local# pip3 -V
pip 9.0.1 from /usr/lib/python3/dist-packages (python 3.6)
```

I'm running an image of Ubuntu `gcr.io/gcp-runtimes/ubuntu_18_0_4:latest`

I get the error while running the following:
`pip3 -r requirements.txt`

requirements.txt:
```
-e git+https://github.com/Bouke/django-two-factor-auth#egg=two_factor
```

when checking my system locale:
```
root@d1a4620dbff1:/usr/local# locale                                                                                                                                                                                                                                      LANG=                                                                                                                                                                                                                                                                            LANGUAGE=                                                                                                                                                                                                                                                                        LC_CTYPE="POSIX"                                                                                                                                                                                                                                                                 
LC_NUMERIC="POSIX"                                                                                                                                                                                                                                                               
LC_TIME="POSIX"                                                                                                                                                                                                                                                                  
LC_COLLATE="POSIX"                                                                                                                                                                                                                                                               
LC_MONETARY="POSIX"                                                                                                                                                                                                                                                              
LC_MESSAGES="POSIX"                                                                                                                                                                                                                                                              
LC_PAPER="POSIX"                                                                                                                                                                                                                                                                 
LC_NAME="POSIX"                                                                                                                                                                                                                                                                  
LC_ADDRESS="POSIX"                                                                                                                                                                                                                                                               
LC_TELEPHONE="POSIX"                                                                                                                                                                                                                                                             
LC_MEASUREMENT="POSIX"                                                                                                                                                                                                                                                           
LC_IDENTIFICATION="POSIX"                                                                                                                                                                                                                                                        
LC_ALL= 
```

**Solution:**
I had to change my locale settings to UTF-8, then the install worked.
```
root@d1a4620dbff1:/usr/local# locale -a                                                                                                                                                                                                                                   
C                                                                                                                                                                                                                                                                                
C.UTF-8                                                                                                                                                                                                                                                                          
POSIX                                                                                                                                                                                                                                                                            
root@d1a4620dbff1:/usr/local# export LC_ALL=C.UTF-8 
```

**Comment by Bouke** [comment]
I've added a fix for this in 0f5c6a5242e8ef80733eac726c2a4ce6f37e69d4.

**Comment by pham7quang** [comment]
Thanks so much @Bouke!
